src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -99,6 +99,22 @@ export type MultiComboboxProps<T> = {
   canClearAllAfter?: number;
 };
 
+function flattenObject(ob: Array<any> | { [k: string]: any }): string[] {
```

Comment:
I'm not sure whether the name conveys exactly what this function doees, since it creates dot-separated key paths.

I've also seen this:

```tsx
flattenObject({"x": { "y": [1,2,3]}})
// ["x.y", "x.y", "x.y"]
```

which I think is a bug.

Ultimately, if we do want this sort of behaviour, thihs should be panther-specific and not pounce-specific. What I'd propose is to allow `onPaste` or `processUserValue` to be a prop that can be customized (both would default to what we currently have) and then handle it on the panther-enterprise side.

Generally, pasting JSON into a tokenfield isn't expected, so I'm hesitent to mark this behaviour as "bug". What I do understand though, is that on a particular field in the alert summaries, you want to get a key path from a JSON. That's why I think that pounce should offer the pasting/processing customization capability and panther-enterprise should handle it as it sees fit